cmd/minishift/cmd/config/view.go

```diff
@@ -89,9 +92,9 @@ func configView() error {
 	}
 
 	lines := strings.Split(buffer.String(), "\n")
-	// remove empy line at end
+	// remove empty line at end
 	lines = lines[:len(lines)-1]
 	sort.Strings(lines)
-	fmt.Print(strings.Join(lines, "\n"))
+	fmt.Println(strings.Join(lines, "\n"))
```

Comment:
There are two issues. One about the output format and one about the ability to use the --format flag of the command